contents/blog/what-is-a-growth-engineer.md

```diff
@@ -0,0 +1,136 @@
+
+3. **Data-driven.** More than other engineering roles, they care about usage data and it guides their work. They use data to inform and evaluate all the experiments they do. They need data, and the ability to analyze that data, to know which ideas to work on and how their solutions are working.
+
+4. **Curious.** Although it sounds a bit cliche, curiosity enables growth engineers to discover new opportunities, grasp new technologies, and figure out whatever they need to drive growth. They work with more tools and technologies than traditional engineers do. This means more learning, and being unafraid to dive into new subjects.
+
+At PostHog, we find these characteristics often arise in former technical founders, and that means our growth team has always been entirely made up of them. Founders understand what it takes to build a product, have a combination of engineering and business skills, and are willing to figure out anything to drive growth.
+
+## What skills does a growth engineer need?
+
+Growth engineers are responsible for three key tasks:
+
+1. Creating ideas and strategies for growth.
+2. Implementing those ideas and strategies.
+3. Measuring and evaluating those implementations.
+
+To accomplish these tasks, they need a combination of engineering, business, product, design, and data skills.
+
+### Engineering skills
+
+Growth engineers write code to implement their ideas and experiments. If they couldn’t code, they wouldn’t be an engineer, and that is half the name. They must be comfortable across the stack, use new tools like external APIs, and work we unfamiliar tech if necessary.
+
+> **Growth engineers vs growth hackers:** Growth hackers also aim to drive growth but focus on more non-product, non-technical solutions. Growth engineering, for some teams, might include this, but for us, the role focuses on technical work across products.
+
+### Product and design knowledge
+
+Second, the other part of creating and implementing ideas is **product and design knowledge**. Because a growth team works across the stack and teams, they are often "lone wolves." They need the ability to fend for themselves in product and design decisions. They don’t have to be Da Vinci, but this knowledge helps them create successful features.
+
+As [Porter](https://www.workatastartup.com/jobs/46166), a YC-backed devtool company founded in 2020, says:
+
+> We're looking for a product engineer with strong design chops who is fluent in React/Typescript and CSS. Simply put, we want someone who will take Porter's usability and UX/visual polish to the next level. You will be involved in all feature discussions and will shape the trajectory of our product.
```

Comment:
It stood out for me that this said Product engineer rather than growth engineer

Reply:
Ahh, it's a growth/product engineer role 🤔